openquakeplatform_standalone/utils.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from importlib import import_module
+
+def oq_context_processor(request):
+    """
+    A custom context processor which allows injection of additional
+    context variables.
+    """
+
+    context = {}
+
+    context['app_list'] = []
+
+    cl_list = [ 'calc', 'share', 'explore' ]
+    ct = 0
+    for app in settings.STANDALONE_APPS:
+        # remove 'openquakeplatform_' suffix with slicing
+        app_name = app[18:]
```

Comment:
To be consistent with names with underscores we must increase the complexity of the operation.
Better keep the original version.